Source/API/EbApi.h

```diff
@@ -38,6 +38,33 @@ extern "C" {
 #define EB_NON_REF_PICTURE   4
 #define EB_INVALID_PICTURE   0xFF
 
+#define EB_SEGMENT_BLOCK_SIZE 64
+    typedef struct
+    {
+        uint8_t ovFlags;
+        int8_t  qpOv;
+        int8_t  deblockOv;
+        int8_t  filterOv;
+    } SegmentOverride_t;
+
+    typedef enum EB_OV_FLAGS
+    {
+        /* Do no ovveride default values */
+        EB_NO_OV              = 0,
+        /* Set qp value in range [0:51] */
+        EB_QP_OV_DIRECT       = 1 << 0,
+        /* Change original qp value in range [-26:25] */
+        EB_QP_OV_DELTA        = 1 << 1,
+        /* Use qpOv for deblock, behaviour depends on EB_QP_OV* flag */
+        EB_DENSITY_QP_OV      = 1 << 2,
+        /* Use deblockOv for deblock in range [-25:25];
+        * positive value decrease density, negative increase */
+        EB_DENSITY_DEBLOCK_OV = 1 << 3,
+        /* Change first stage of quantization in range [-7;7];
+        * positive value reduce bitrate, negative increase */
+        EB_TU_FILTER_OV       = 1 << 4,
```

Comment:
Setting the 2 ones above for the bitmap hasn't been completed yet?

Reply:
Fixed by implementation of 2nd idea ("SegmentOvFile")